user_guide_src/source/libraries/uploaded_files.rst

```diff
@@ -126,7 +126,7 @@ request, use ``getFiles()``. This will return an array of files represented by i
 Of course, there are multiple ways to name the file input, and anything but the simplest can create strange results.
 The array returns in a manner that you would expect. With the simplest usage, a single file might be submitted like::
 
-    <input type="file" name="avatar" />
+    <input type="file" name="avatar">
```

Comment:
`4.3` improves HTML5 compatibility.
https://github.com/codeigniter4/CodeIgniter4/blob/4.3/user_guide_src/source/changelogs/v4.3.0.rst#html5-compatibility
But 4.2 is not.

Do you really need to these tag changes?

Reply:
Ok, I got it and I have been reverted it.

Reply:
Thank you.